baklava/resources/overlay/asset-manifest.json

```diff
@@ -0,0 +1,22 @@
+{
+  "files": {
+    "main.css": "./static/css/main.f434b18b.chunk.css",
+    "main.js": "./static/js/main.bc92b591.chunk.js",
+    "main.js.map": "./static/js/main.bc92b591.chunk.js.map",
+    "runtime-main.js": "./static/js/runtime-main.7dea6612.js",
+    "runtime-main.js.map": "./static/js/runtime-main.7dea6612.js.map",
+    "static/js/2.b319a4ae.chunk.js": "./static/js/2.b319a4ae.chunk.js",
+    "static/js/2.b319a4ae.chunk.js.map": "./static/js/2.b319a4ae.chunk.js.map",
+    "static/js/3.5b106a42.chunk.js": "./static/js/3.5b106a42.chunk.js",
```

Comment:
don't check in minified js, we can add a build step instead

Reply:
Didn't get you? Do you want me to shift the react project for the overlay into baklava folder itself?

Reply:
If yes then wont that increase the size of baklava like a lot? Cuz the node_modules and stuff?